@ecomplus/storefront-template/template/js/netlify-cms/base-config/collections/widgets.js

```diff
@@ -41,5 +42,6 @@ export default options => ({
     fixWidgetConfig(options, getWidgetTawkto),
     fixWidgetConfig(options, getWidgetTrustvox),
     fixWidgetConfig(options, getWidgetEbit)
+    fixWidgetConfig(options, getWidgetOffersNotification)
```

Comment:
```suggestion
    fixWidgetConfig(options, getWidgetEbit),
    fixWidgetConfig(options, getWidgetOffersNotification)
```

Faltou esta vírgula, e por isto deu erro de sintaxe https://app.netlify.com/sites/storefront-template/deploys/5eee3503004da5000794f61f